source/opcode.cpp

```diff
@@ -236,10 +236,10 @@ int32_t spvOpcodeIsPointer(const SpvOp opcode) {
   switch (opcode) {
```

Comment:
The name of this function is now misleading.  It should be more indicative of its function, which is whether the opcode may produce a pointer (in a particular addressing model).